### PR TITLE
Clarifies git setup for doc contributors

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Specifies the requirement about user name and email (encountered by community member)

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: CONTRIBUTING_DOCS.md [hunk 1]
```diff
@@ -61,6 +63,8 @@ The submission of a PR kicks off a continuous integration process which includes
 
 However, you can also run this after submitting your PR and experiencing an `htmlproofer` failure from the Semaphore job.
 
+> **Note**: Sometimes `htmlproofer` may deliver errors that do not reproduce. If you get an error from `htmlproofer` that seems incorrect, rerun `htmlproofer`. In addition, sometimes the Semaphore job will return spurious `htmlproofer` errors. In this event, one of the maintainers can kick the job back off again.
```
Comment (bcreane) [inline CONTRIBUTING_DOCS.md]: suggest "...can restart the job." in place of '...can kick the job back off again.'

Comment (emanic) [reply CONTRIBUTING_DOCS.md]: Yes, good point. I think it's better.

File: CONTRIBUTING_DOCS.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ Instead of filing a GitHub issue, consider making a PR instead. You are likely t
 
 The doc contribution process works as follows.
 
+> **Prerequisite**: To sign our contributer license agreement (required), your local git must be [configured with a user name and email address](https://help.github.com/articles/setting-your-commit-email-address-in-git/) that match those of your GitHub account.
```
Comment (ozdanborne) [inline CONTRIBUTING_DOCS.md]: Seems odd to have a prerequisite presented as a blockquote. Remove the `>`?

File: CONTRIBUTING_DOCS.md [hunk 3]
```diff
@@ -61,6 +63,8 @@ The submission of a PR kicks off a continuous integration process which includes
 
 However, you can also run this after submitting your PR and experiencing an `htmlproofer` failure from the Semaphore job.
 
+> **Note**: Sometimes `htmlproofer` may deliver errors that do not reproduce. If you get an error from `htmlproofer` that seems incorrect, rerun `htmlproofer`. In addition, sometimes the Semaphore job will return spurious `htmlproofer` errors. In this event, one of the maintainers can restart the job.
```
Comment (ozdanborne) [inline CONTRIBUTING_DOCS.md]: I'd rather we fix the root of this problem than just accept that the test is flaky. Let's remove this disclaimer and set aside some time for someone (could be me) to actually investigate and resolve this.

**Pull request merged**